Data/Vector.hs

```diff
@@ -382,6 +383,26 @@ instance MonadZip Vector where
   {-# INLINE munzip #-}
   munzip = unzip
 
+-- | @since 0.13.0.0
```

Comment:
It would be nice to comment on the semantics of `MonadFix Vector`. It is similar to `MonadFix []` instance, isn't it? If it is, then what's wrong with definining it as below?
```
mfix f = V.fromList (mfix (V.toList . f))
```
I mean, the proposed implementation already `cons`es elements one by one, which is probably not very efficient.

Reply:
The proposed implementation does *not* cons elements one by one. It only uses `cons` *once*, and that `cons` should fuse with the following `generate`. A comment would be in order. Yes, semantics are the same as the `[]` instance, or close enough.

Reply:
The listy implementation could go in the instance documentation if you like.

Reply:
Ah, sorry, my bad, you are right. 
Yes, adding the "listy" implementation in the instance documentation sounds good to me.